src/Orchard.Web/Modules/Orchard.DynamicForms/Drivers/TextFieldElementDriver.cs

```diff
@@ -79,7 +80,7 @@ public class TextFieldElementDriver : FormsElementDriver<TextField>{
         protected override void OnDisplaying(TextField element, ElementDisplayContext context) {
             context.ElementShape.ProcessedName = _tokenizer.Replace(element.Name, context.GetTokenData());
             context.ElementShape.ProcessedLabel = _tokenizer.Replace(element.Label, context.GetTokenData());
-            context.ElementShape.ProcessedValue = _tokenizer.Replace(element.RuntimeValue, context.GetTokenData());
+            context.ElementShape.ProcessedValue = HttpUtility.HtmlDecode(_tokenizer.Replace(element.RuntimeValue, context.GetTokenData()));
```

Comment:
The actual fix is to not encode it in the first place, by using the correct `Replace` overload. But please check with @sfmskywalker if this is the correct behavior. Or anyone who could confirm.